developer/customization/articles/02-architecture/03-understanding-services/02-osgi-cdi-integration/00-osgi-cdi-integration-intro.markdown

```diff
@@ -0,0 +1,36 @@
+# OSGi CDI Integration 
+
+OSGi CDI Integration brings features and capabilities of Contexts and Dependency
+Injection (CDI) to OSGi and makes OSGi services available to beans.  Here's what
+you'll learn how to do:
+
+- Publish CDI beans as OSG services: Register CDI beans as services you can use 
+to customize @product@ components. 
+
+- Use OSGi services in beans: Leverage any OSGi service published on @product@ 
+in any bean. 
+
+The following use cases provide more detail. 
+
+## Use Case: Registering a CDI bean as an OSGi service 
+
+As a CDI bean developer, you want to use a CDI bean to customize or override an
```

Comment:
can we change `customize` with `provide`?

Reply:
Changed `customize` to `provide`.